### PR TITLE
Fixes the CHECK_URLs list

### PR DESCRIPTION
**Pull request by bwhaley**
Fixes the CHECK_URLS input to (a) be applied to the correction function, and (b) be serialized as CSV from Terraform => Python.

### CODE REVIEW

File: functions/replace-route/app.py [hunk 1]
```diff
@@ -358,7 +358,7 @@ def connectivity_test_handler(event, context):
     logger.info("Starting NAT instance connectivity test")
 
     check_interval = int(os.getenv("CONNECTIVITY_CHECK_INTERVAL", DEFAULT_CONNECTIVITY_CHECK_INTERVAL))
-    check_urls = os.getenv("CHECK_URLS", DEFAULT_CHECK_URLS)
+    check_urls = os.getenv("CHECK_URLS", DEFAULT_CHECK_URLS).split(",")
```
Comment (arkadiyt) [inline functions/replace-route/app.py:361]: You need to update `DEFAULT_CHECK_URLS` also

Comment (bwhaley) [reply functions/replace-route/app.py]: Monday after Sunday of re:Invent is always rough.

**Review by alindeman — APPROVED** [review]
Nice

**Comment by bwhaley** [comment]
Thanks for the sanity check and reviews. Mondays, amirite?

**Pull request merged**